fruit_classifier/predict/predict_utils.py

```diff
@@ -96,3 +98,33 @@ def load_classifier():
     model = load_model(str(model_path))
 
     return model
+
+
+def pick_random_image(from_folder):
+    """
+    Picks a random image from a random directory in "from_folder"
+
+    Note, this includes images from both training set and test set.
+
+    Parameters
+    ----------
+    from_folder : str - the folder where an image is found
```

Comment:
Please use numpy docstring

```suggestion
    from_folder : str
        The directory where an image is found
```

Reply:
Also, it looks like this will be relative to the `generated_data` directory. If so this should be documented